cranelift/codegen/src/isa/aarch64/inst/args.rs

```diff
@@ -152,6 +174,14 @@ impl MemArg {
         MemArg::UnsignedOffset(reg, UImm12Scaled::zero(I64))
     }
 
+    /// Return a reg if this memory address is just a register with no other offset.
+    pub fn is_reg_no_offset(&self) -> Option<Reg> {
```

Comment:
This is actually unused in this PR, so it could be removed? (unless you plan to use it soon later)

(`pub(crate)` would show you a warning if it's effectively unused)

Reply:
Removed, thanks!